src/overlays/actors/ovl_Demo_Getitem/z_demo_getitem.h

```diff
@@ -3,13 +3,17 @@
 
 #include "global.h"
 
+#define DEMOGETITEM_GET_F(thisx) ((thisx)->params & 0xF)
+
 struct DemoGetitem;
 
 typedef void (*DemoGetitemActionFunc)(struct DemoGetitem*, PlayState*);
 
 typedef struct DemoGetitem {
     /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x8];
+    /* 0x144 */ s16 item;
+    /* 0x146 */ u16 action;
```

Comment:
```suggestion
    /* 0x146 */ u16 csAction;
```
To avoid confusion with this sort of thing: https://github.com/zeldaret/oot/blob/master/src/overlays/actors/ovl_En_Zf/z_en_zf.c#L642

Reply:
i didn't get what you tried to tell me with that link.

Reply:
The main thing is there is a difference between `action`s and `cutscene actions` so its best to be specific here by prepending `cs` for cutscene.